### DISCUSSION TITLE
Responsive margins.

### DISCUSSION THREAD

**Discussion by doug123** [wet-boew/wet-boew#9407] [Q&A]
Hi there,

I need to implement responsive margins, but this doesn't appear that this is supported by WET OOTB though.
https://wet-boew.github.io/wet-boew-styleguide/design/proximity-en.html

I am comfortable writing my own CSS to accomplish this, but before I do, I am curious how others are tackling this issue?

Any thoughts are appreciated.

Thank you.

**Comment by ghost** [comment]
This project uses Bootstrap 3.4.1 which did not have this.

Bootstrap 4 and 5 did eventually add this. I take a look at their approach.

CSS margins added to Bootstrap 3 by this project
https://github.com/wet-boew/wet-boew/blob/d717897e574217a19099a5ed72145a77a2d0cc36/src/base/proximity/_base.scss
